styles/XH.scss

```diff
@@ -51,7 +51,7 @@ body.xh-app {
     margin: var(--xh-pad-px) 0;
   }
 
```

Comment:
Seems like this block could be moved to kit/blueprint/styles.scss since it deals specifically with blueprint overrides.